patterns/2-structured/trusted-committer.md

```diff
@@ -6,51 +6,49 @@ Trusted Committer
 
 Many InnerSource projects will find themselves in a situation where
 they consistently receive feedback, features, and bug-fixes from contributors.
-In these situations project maintainers seek ways to recognize and reward the
+In these situations, project maintainers seek ways to recognize and reward the
 work of the contributor above and beyond single contributions.
 
 ## Problem
 
 - Project maintainers want to find ways to scale their ability to support a project
 - Project maintainers want to find ways to lengthen the value delivered by a project
 - Project maintainers want to visibly reward frequent contributors and empower them to amplify their value contribution.
-- Lack of language for recognizing contributions across teams within an organization
+- Lack of mechanism for recognizing contributions across teams within an organization
```

Comment:
```suggestion
- Lack of mechanism and language for recognizing contributions across teams within an organization
```
 
I agree that the TC concept overall is more of a mechanism, as it comes with expectations, process, etc.
However I do think that the introduction of the term "Trusted Committer" into the shared language of a project is also an essential piece of the change. Therefore I would opt for keeping both.